projects/canopy/src/lib/details/details-panel-heading/details-panel-heading.component.ts

```diff
@@ -47,13 +45,11 @@ export class LgDetailsPanelHeadingComponent {
   @Output() toggleActive = new EventEmitter<boolean>();
 
   chevronDown = lgIconChevronDown.name;
-  id = nextUniqueId++;
-  toggleId = `lg-details-header-${this.id}`;
-  panelId = `lg-details-content-${this.id}`;
+  uniqueId: number;
```

Comment:
you never set `uniqueId`to anything, as far as i can see

Reply:
I set it in the parent component using the `@ContentChild` reference